docs/docs/recipes/protect-your-api/README.md

```diff
@@ -257,10 +257,117 @@ Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3O
+
+### Get the JWK Public Key
+
+All tokens issued by logto are signed with [JWK](https://datatracker.ietf.org/doc/html/rfc7517). ( See [JWS](https://datatracker.ietf.org/doc/html/rfc7515) for details)
+You will need a public key to verify the signature of the Bearer Token(_access_token_) before moving on.
+
+The key set can be inquired through Logto's **jwks_uri**: `https://<your-logto-domain>/oidc/jwks`.
+
+:::note
+All the latest Logto Authorization Server Configurations can be found by `https://<your-logto-domain>/oidc/.well-known/openid-configuration`. Including the **jwks_uri** and other authorization configs.
+:::
+
+As a response, you will get a JWKS ( JSON Web Key Sets). **Keep it** on your server. You will need it to verify all the requests' token signatures.
+
+### Validate the existence and extract the Bearer Token from Headers
+
+The request should contains a `Authorization` header with a `Bearer <access_token>` format. Extract the Bearer Token from request header:
+
+<Tabs>
+
+<TabItem value="js" label="NodeJs">
+
+```js
+import { IncomingHttpHeaders } from 'http';
+
+const extractBearerTokenFromHeaders = ({ authorization }: IncomingHttpHeaders) => {
+  assertThat(authorization, new Error({ code: 'auth.authorization_header_missing', status: 401 }));
+
+  assertThat(
+    authorization.startsWith('Bearer'),
+    new Error({ code: 'auth.authorization_token_type_not_supported', status: 401 })
+  );
+
+  return authorization.slice(bearerTokenIdentifier.length + 1);
+};
+```
+
+</TabItem>
+<TabItem value="java" label="java">
+// TODO:
+</TabItem>
+
+</Tabs>
+
+### Token Validation
+
+A encoded [JWS](https://datatracker.ietf.org/doc/html/rfc7515) token is constructed with three parts:
+
+- JOSE Header: Declares the code type and encode algorithm
+- JWS Payload: Includes all the token's claims
+- JWS Signature: Signature signed with JWK
+
+A standard schema of Logto JWS Payload: (claims may very, based on your OIDC config)
+
+| key       | description                |
+| --------- | -------------------------- |
+| jti       | unique JWT ID              |
+| sub       | subject, usually user id   |
+| iat       | timestamp token issues at  |
+| exp       | timestamp token expires at |
+| client_id | application id             |
+| iss       | token issuer identity      |
+| aud       | audience of the token      |
+
+:::note
+For development, you could use a handy site, [jwt.io](https://jwt.io/), to decode and check the tokens you received. Be careful with the tokens come from production env, as the service is provided by a third party, and your token may be toasted.
+:::
+
+The Bearer Token should be accepted only if:
+
+- the token's signature is verified
+- the token's issuer is `https://<your-logto-domain>/oidc` (issued by the Logto server)
+- the token's audience is the current API indicator (audience-restricted)
+- the token is with n its valid lifetime
```

Comment:
is this a standard process? do we have link to reduce the content size?

Reply:
https://datatracker.ietf.org/doc/html/rfc7519#section-7.2 only provides a standard JWT format validation guide. Need to mention the payload validation here.